### PR TITLE
Adds validation fixes according to WordPress CSS coding standards

### PR DESCRIPTION
**Pull request by marcustisater**
I went through all the selectors property values to match the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/). One thing that I noticed is that comments do not follow the best practices and sometimes it's unorganized, I will created an issue about this a bit more in-depth.

I would also like to point out that there are a lot of heavy selectors in this stylesheet that will make it more difficult for developers to override changes, also something we should strongly consider refactoring. 

Happy to help, will try to get myself involved more into refactoring the CSS.

### CODE REVIEW

File: style.css
Comment (ntwb) [inline style.css]: Although not a requirement the suggestion is to maintain CamelCase:

> via https://www.w3.org/TR/CSS22/ui.html#system-colors
> "Although these are case-insensitive, it is recommended that the mixed capitalization shown below be used, to make the names more legible."

WordPress core doesn't use any system colors currently so there's no "prior art" in this regard, I think we should use the more broadly used CamelCase here though.

File: style.css
Comment (ntwb) [inline style.css]: I think this should remain `bolder`

In a more broader scenario I think the WordPress CSS Coding Standards should allow the use of `bolder` and `lighter` relative weights

File: style.css
Comment (ntwb) [inline style.css]: Expected double quotes

File: style.css
Comment (ntwb) [inline style.css]: Expected double quotes

File: style.css
Comment (ntwb) [inline style.css]: A single space should follow every comma `,`, i.e. `rgba(0, 0, 0, 0)` not `rgba(0,0,0,0)`

File: style.css
Comment (ntwb) [inline style.css]: A single space should follow every comma `,`, i.e. `rgba(0, 0, 0, 0)` not `rgba(0,0,0,0)`

File: style.css
Comment (ntwb) [inline style.css]: A single space should follow every comma `,`, i.e. `rgba(0, 0, 0, 0)` not `rgba(0,0,0,0)`

File: style.css
Comment (ntwb) [inline style.css]: A single space should follow every comma `,`, i.e. `rgba(0, 0, 0, 0)` not `rgba(0,0,0,0)`

File: style.css
Comment (ntwb) [inline style.css]: A single space should follow every comma `,`, i.e. `rgba(0, 0, 0, 0)` not `rgba(0,0,0,0)`

File: style.css
Comment (ntwb) [inline style.css]: A single space should follow every comma `,`, i.e. `rgba(0, 0, 0, 0)` not `rgba(0,0,0,0)`

**Review by ntwb — CHANGES_REQUESTED** [review]
@marcustisater This is pretty close to perfect, I've added some notes for review.

File: style.css
Comment (ntwb) [inline style.css]: This should be indented 1 tab, not 2 spaces

File: style.css
Comment (akshayvinchurkar) [reply style.css]: for better readability you are right.

File: style.css
Comment (akshayvinchurkar) [reply style.css]: keep the same indentation in all rules

**Comment by marcustisater** [comment]
Thanks @ntwb, I will go through the notes. I wasn't really sure about the gradient stuff.

**Comment by ntwb** [comment]
@marcustisater, yeah, I've got a follow up issue to create for the gradients, I'll do that as a separate issue though, as it's more a bug than a coding standard change.

**Comment by marcustisater** [comment]
Thanks Stephen, I agree with you. I went through your notes but the only thing I didn't quite understand was the double quotes part for the gradient, I am not used to writing gradients... :joy:

**Comment by ntwb** [comment]
@marcustisater change the single quote `'` to double quote `"` e.g:

From: `filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#4d000000', GradientType=0); /* IE6-9 */`

To: `filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#00000000", endColorstr="#4d000000", GradientType=0); /* IE6-9 */`

**Comment by ntwb** [comment]
This https://github.com/WordPress/twentyseventeen/pull/169/commits/9218f4bf005de8cde56989ddbdf74345dadc3ecd commit should NOT be part of this PR.

**Comment by marcustisater** [comment]
Sorry, bad habit.

All good now? It should remove 9218f4b from this PR now or do I need to make a separated request? It's not in my fork.

File: style.css
Comment (ntwb) [inline style.css]: Expected single space after `,` (The middle set of RGB values)

File: style.css
Comment (ntwb) [inline style.css]: Expected single space after `,` (Same middle set of RGB values)

**Review by ntwb — CHANGES_REQUESTED** [review]
Added a couple of notes for two issue of space after comma

**Comment by marcustisater** [comment]
Thanks, that must have slipped by... :blush:

**Review by ntwb — APPROVED** [review]
Looks good 👍🏻

**Comment by marcustisater** [comment]
@davidakennedy look through and merge, then we can get on to #194. Thanks for the review once again @ntwb

**Comment by marcustisater** [comment]
@melchoyce I resolved the conflicts, can be merged.

**Comment by davidakennedy** [comment]
@marcustisater Thanks for the work here – sorry this didn't get merged sooner. Closing this out as I think most of this has been taken care of in other PRs. Thanks for the reviews @ntwb!

**Comment by ntwb** [comment]
The fast pace changes reflected in `style.css` made this PR tricky to say the least, wait for a review, make changes, rebase, make more changes, wait for another review, rebase, rinse and repeat 😖 

That said I'm still seeing 100+ CSS coding standards issues, there is probably 2 ways to go away fixing these:
1. Small pull requests updating only a small set of changes, e.g A PR switching from named font weight values to numeric, then another PR removing trailing zeros from values etc, that should allow for quicker code review and hopefully not need rebasing before merge etc.
2. I can submit a PR making all the changes in one swoop at a time organised in Slack where it can be discussed and merged before other PRs cause merge conflicts etc

I'll also echo @davidakennedy comment @marcustisater, top work here and thanks 👍

**Comment by marcustisater** [comment]
ok no worries.

**Pull request closed without merging**
